gcp/workers/alias/upstream_computation.py

```diff
@@ -155,12 +156,12 @@ def main():
   # and avoid datastore emulator problems, see issue #2093
   updated_bugs = []
   logging.info('Retrieving bugs...')
-  bugs = osv.Bug.query(
-      ndb.OR(osv.Bug.upstream_raw > '', osv.Bug.upstream_raw < ''))
-  bugs = {
-      bug.db_id: bug
-      for bug in bugs.iter(projection=[osv.Bug.db_id, osv.Bug.upstream_raw])
-  }
+  bugs_query = osv.Bug.query(osv.Bug.upstream_raw > '')
+
+  bugs = defaultdict(list)
+  for bug in bugs_query.iter(projection=[osv.Bug.db_id, osv.Bug.upstream_raw]):
+    bugs[bug.db_id].append(bug.upstream_raw[0])
+  bugs = {k: list(set(v)) for k, v in bugs.items()}
```

Comment:
There's actually quite a bit of conversion between `list` and `set` going on here and in `compute_upstream`

I think you might be able to get away with using only sets throughout (use `defaultdict(set)` instead)

Reply:
Oh this has changed since I wrote the comment?